### PR TITLE
feat: firebase setup

### PR DESCRIPTION
**Pull request by heypran**
### Changes

- [x] Setup firebase 

### Affected Platforms

- [ ] Android
- [ ] iOS
- [x] Web

### Tested on Platforms

- [ ] Android
- [ ] iOS
- [ ] Web

### Checks

- [x] Build

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +25,7 @@
     "**/*": "prettier --write --ignore-unknown"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "APP_ENV=dev react-scripts start",
```
Comment (sinharaksh1t) [inline package.json:28]: remove...
```suggestion
    "start": "react-scripts start",
```

**Review by sinharaksh1t — CHANGES_REQUESTED** [review]

**Comment by sinharaksh1t** [comment]
Also, remember to add the dev dependencies.

File: package.json (hunk 1 above)
Comment (heypran) [reply package.json:21]: done

**Comment by heypran** [comment]
Thanks @sinharaksh1t

**Review by sinharaksh1t — APPROVED** [review]

**Pull request merged**